prov/shm/src/smr_progress.c

```diff
@@ -346,7 +356,8 @@ static int smr_mmap_peer_copy(struct smr_ep *ep, struct smr_cmd *cmd,
 	return ret;
 }
 
-static int smr_progress_mmap(struct smr_cmd *cmd, struct iovec *iov,
+static int smr_progress_mmap(struct smr_cmd *cmd, enum fi_hmem_iface iface,
+			     uint64_t device, struct iovec *iov,
```

Comment:
Same comment as above. 
I know we did a bad job before your change, let's unify it since you are here.